modules/templates/templates_api.rb

```diff
@@ -25,4 +25,11 @@ class Proxy::TemplatesApi < Sinatra::Base
       Proxy::Templates::TemplateProxyRequest.new.get_template([kind].join('/'), request.env, params)
     end
   end
+
+  post "/:kind" do |kind|
+    log_halt(500, "Failed to retrieve #{kind} template for #{params.inspect}: ") do
```

Comment:
Are we actually trying to retrieve template here? Perhaps more generic message would be a bit better